samples/subsys/mgmt/mcumgr/smp_svr/boards/tlsr9518adk80d.conf

```diff
@@ -0,0 +1 @@
+CONFIG_IMG_BLOCK_BUF_SIZE=256
```

Comment:
I can't access the datasheet for this part, is there any particular reason that this needs to be set to 256 instead of the default 512?

Reply:
Hi @lairdjm 

Thanks for nice question. The answer is: b91 platform has 256 bytes flash page size.

Just for curiosity, I've check it once again and remove the config (which means that IMG_BLOCK_BUF_SIZE has default value 512). **mcumgr** image update operation hung immediately after start. The reason is that function **find_flash_total_size** (stream_flash.c) return false on the condition `if (ctx->buf_len > info->size)`. Also I see log message "Buffer size is bigger than page"

Reply:
@de-nordic Should this go in the boards folder in Kconfig.defconfig with an if around it or in the mcumgr sample? Seems like it would be applicable to many samples and end applications

Reply:
Hi @lairdjm, @de-nordic 

The config is moved into tlsr9518adk80d_defconfig with detailed comment. Please, take a look and approve the PR if everything is ok.

Regards,
Alex